algorithms/searching/breadth_first_search.py

```diff
@@ -0,0 +1,24 @@
+"""
+Author: Rafeh Qazi (Rafeh01 on github)
+Iterative queue based implementation of a BFS algorithm on a graph.
+Breadth First Search Overview:
+    Used to traverse graphs or trees.
+    Starts at a selected node and searches all the closest connected
+    nodes first before moving on to the next nodes.
+"""
```

Comment:
Can you match the doc strings found in other modules? For an example take a look at: https://github.com/nryoung/algorithms/blob/master/algorithms/sorting/strand_sort.py#L1-L18